Purchases/Public/Purchases.swift

```diff
@@ -290,9 +335,9 @@ public typealias DeferredPromotionalPurchaseBlock = (@escaping PurchaseCompleted
          offeringsManager: OfferingsManager,
          purchasesOrchestrator: PurchasesOrchestrator) {
 
-        Logger.debug(Strings.configure.debug_enabled)
-        Logger.debug(Strings.configure.sdk_version(sdkVersion: Self.frameworkVersion))
-        Logger.user(Strings.configure.initial_app_user_id(appUserID: appUserID))
+        Logger.debug(Strings.configure.debug_enabled, fileName: nil)
+        Logger.debug(Strings.configure.sdk_version(sdkVersion: Self.frameworkVersion), fileName: nil)
+        Logger.user(Strings.configure.initial_app_user_id(appUserID: appUserID), fileName: nil)
```

Comment:
why set the file name to `nil` here?

Reply:
These are the initial logs on initialization and they become unnecessarily verbose with this. I thought it was the one log which we don't care where it comes from (see sample log in the PR description)